src/Refactoring-Core/RBIsNotGlobal.class.st

```diff
@@ -0,0 +1,32 @@
+"
+I check if no global in the `model` is named the same as `name`
+"
+Class {
+	#name : 'RBIsNotGlobal',
+	#superclass : 'RBVariableNameCondition',
+	#instVars : [
+		'model'
+	],
+	#category : 'Refactoring-Core-Conditions',
+	#package : 'Refactoring-Core',
+	#tag : 'Conditions'
+}
+
+{ #category : 'instance creation' }
+RBIsNotGlobal class >> name: aString model: aNavigationModel [
+
+	^ (self name: aString)
+		  model: aNavigationModel;
+		  yourself
+]
+
+{ #category : 'checking' }
+RBIsNotGlobal >> check [
+
+	^ (model includesGlobal: name asSymbol) not
+]
+
```

Comment:
We could define violators to return an empty array or an array with the faulty one. 
Now sure. 
Now we need some tests for these new conditions.